packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -270,6 +270,15 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   bool get haveDimensions => _haveDimensions;
   bool _haveDimensions = false;
 
+  /// Whether scrollables should absorb pointer events at this position.
```

Comment:
@Michal-MK if this doc adjustment LGT-you, let me know and we can land this change.